src/BuddyManager.cpp

```diff
@@ -651,6 +661,20 @@ void BuddyManager::emailSend(CNSocket* sock, CNPacketData* data) {
 
     // handle items
     std::vector<sItemBase> attachments;
```

Comment:
Total nitpick, but the way you placed your check separates the vector declaration from the loop that fills it. It's not absolutely necessary, but it would be nice to keep them together.

Reply:
I have moved my check above the vector declaration.